packages/@aws-cdk/aws-cloudformation/package.json

```diff
@@ -57,7 +57,7 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@aws-cdk/assert": "^0.14.1",
-    "@types/lodash": "^4.14.116",
+    "@types/lodash": "^4.14.117",
```

Comment:
Intentional?

Reply:
Yes. I forgot to mention it in the PR description... #oops